### PR TITLE
Issue #1064 wel from imod

### LINKED ISSUE #1064: Implement `from_imod5_data` method to Well package, requires DIS and NPF package as arguments

### PR DESCRIPTION
**Pull request by luitjansl**
Fixes #1064 

# Description
Imports wells as grid-agnostic well packages, based on filter top and filter bottom, and x/y location as stated in the imod5_dataset object. 

# Checklist
- [X] Links to correct issue
- [ ] Update changelog, if changes affect users
- [X] PR title starts with ``Issue #nr``, e.g. ``Issue #737``
- [X] Unit tests were added
- [ ] **If feature added**: Added/extended example

### CODE REVIEW

File: imod/mf6/wel.py [hunk 1]
```diff
@@ -609,6 +609,47 @@ def mask(self, domain: GridDataArray) -> Well:
...
+        key: str,
+        imod5_data: dict[str, dict[str, GridDataArray]],
+        minimum_k: float = 0.1,
+        minimum_thickness: float = 1.0,
+    ) -> "Well":
+        wel_x = cls.panda_column_to_scalar(imod5_data[key]["dataframe"]["x"])
+        wel_y = cls.panda_column_to_scalar(imod5_data[key]["dataframe"]["y"])
+        wel_top = cls.panda_column_to_scalar(imod5_data[key]["dataframe"]["filt_top"])
+        wel_bot = cls.panda_column_to_scalar(imod5_data[key]["dataframe"]["filt_bot"])
+        rate = imod5_data[key]["dataframe"]["rate"]
```
Comment (JoerivanEngelen) [inline imod/mf6/wel.py:624]: I don't think the right thing happens here:

x, y, filt_top, and filt_bot can all be arrays. If wells are transient, a transient rate has to be generated with dimensions ("time", "index").

Wells at multiple locations can be part of one Well package.

File: imod/mf6/wel.py [hunk 2]
```diff
@@ -609,6 +609,47 @@ def mask(self, domain: GridDataArray) -> Well:
...
+
+    @classmethod
+    def panda_column_to_scalar(cls, column: pd.Series) -> float:
+        # input is a column of a pandas dataframe expected to contain the same value
+        # in every row (like the x-coordinate of a well)
+        # This function returns that value and checks they are all the same
+
+        if not np.all(column == column[0]):
+            raise ValueError("error while converting pandas column to scalar")
+        return float(column[0])
```
Comment (JoerivanEngelen) [inline imod/mf6/wel.py:651]: This shouldn't be necessary

**Review by JoerivanEngelen — CHANGES_REQUESTED** [review]

File: imod/mf6/wel.py (hunk 2 above)
Comment (luitjansl) [reply imod/mf6/wel.py]: removed this function

File: imod/mf6/wel.py (hunk 1 above)
Comment (luitjansl) [reply imod/mf6/wel.py]: now reworked the PR to account for different wells in the same well package

**Review by JoerivanEngelen — APPROVED** [review]

**Pull request merged**